consensus/src/block_storage/sync_manager.rs

```diff
@@ -83,9 +83,6 @@ impl BlockStore {
         )
         .await?;
 
-        self.insert_quorum_cert(sync_info.highest_commit_cert(), &mut retriever)
```

Comment:
Can you explain why this is not needed?

Reply:
I think we should maintain that all quorum inserted are only for ordering, commit cert should go to the buffer manager. otherwise we may see that ordered cert has version > 0

Reply:
it'll result in this error too
```
2022-07-12T07:12:03.207311Z [consensus] ERROR consensus/safety-rules/src/safety_rules.rs:473 {"error":"Invalid Ordered LedgerInfoWithSignatures: Empty or at least one of executed_state_id, version, or epoch_state are not dummy value: LedgerInfo: [commit_info: BlockInfo: [epoch: 15, round: 3668, id: 1dbf92d4, executed_state_id: 1306ad19, version: 349982, timestamp (us): 1657609921536100, next_epoch_state: None]]","event":"error","name":"sign_commit_vote"}
```